learn/get-started/write-a-graphql-api-with-ballerina.md

```diff
@@ -236,13 +267,15 @@ resource function get filter(string isoCode) returns CovidData? {
 }
 ```
 
-In the above resource method, the `filter` field is defined in the root `Query` type. Since this field has an
-input parameter `isoCode`, you have to add an input parameter to the resource method. This method returns the
-corresponding data for the given `isoCode` if such data is available in our data set, otherwise, it returns `null`.
+In this code:
+- The `filter` field is defined in the root `query` type. Since this field has an input parameter `isoCode`, you have to add an input parameter to the resource method. 
+- This method returns the corresponding data for the given `isoCode` if such data is available in the data set, and it returns `null` otherwise.
+
+#### Create mutation type remote method 
```

Comment:
`mutation type remote method` is not accurate. The remote method is used to define a field inside the `Mutation` type.

Reply:
Updated to `Create `Mutation` type remote function`.